SQL_Assignments/sql_2.sql

```diff
@@ -0,0 +1,137 @@
+Create a table students and insert names in malayalam
+
+    create table employee(empid int primary key,empname nvarchar(20))
```

Comment:
table name employee or student ?

Reply:
the table name should be student that was a careless mistake from my part